server/app/queries/schema.ts

```diff
@@ -6,9 +6,8 @@ CREATE SCHEMA QUIZ_DB;
 
 CREATE TABLE Room (
 	room_id			SERIAL,
-	event_key		CHAR(6)			UNIQUE NOT NULL,
+	event_key		CHAR(8)			UNIQUE NOT NULL,
 	room_name		VARCHAR(64)		NOT NULL,
-	description		VARCHAR(255),
 	presenter_id	VARCHAR(64)		NOT NULL,
 	start_time		DATE			DEFAULT CURRENT_DATE,
```

Comment:
Don't see any functionality value for the start time and default time. While it is not being used, I would recommend commeting out the code and / or removing it entirely

Reply:
I added that for later scalability in case we wanna be able to do Advanced Session Management in MVP+. I can take it off if we won't be using it